src/java/org/apache/cassandra/index/sai/disk/v2/V2VectorIndexSearcher.java

```diff
@@ -165,7 +171,9 @@ else if ((double) cv.getOriginalSize() / cv.getCompressedSize() > 16.0)
     /**
      * Return bit set if needs to search HNSW; otherwise return posting list to bypass HNSW
      */
-    private BitsOrPostingList bitsOrPostingListForKeyRange(QueryContext context, AbstractBounds<PartitionPosition> keyRange, int limit) throws IOException
+    private BitsOrPostingList bitsOrPostingListForKeyRange(QueryContext context,
+                                                           AbstractBounds<PartitionPosition> keyRange,
+                                                           float[] queryVector, int limit) throws IOException
```

Comment:
(formatting convention is either all params on one line, or each param on separate line)